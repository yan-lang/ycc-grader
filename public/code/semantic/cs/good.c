int test()
{
    while(1>2) {
        break;
        break;
        continue;
    }

}