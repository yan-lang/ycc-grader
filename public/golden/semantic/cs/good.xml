<controlStructure>
    <break>
        <line>4</line>
        <attachedLoop>3</attachedLoop>
    </break>
    <break>
        <line>5</line>
        <attachedLoop>3</attachedLoop>
    </break>
    <continue>
        <line>6</line>
        <attachedLoop>3</attachedLoop>
    </continue>
</controlStructure>